### ISSUE TITLE
[yanmaga] Pixel loss when dimension is not multiple of 4

### ISSUE THREAD

**Issue by aihornmac** [shoushouminami/modelpress-downloader#23]
https://github.com/shoushouminami/modelpress-downloader/blob/f7744c08644af3e5ec265f08cb94ee8c876f88c0/src/inject/yanmaga.jp.js#L57-L63

Despite the implementation in source code of `viewer.js`, I found that when dimension is not multiple of 4 (say 1365x2048 as a real example), the remaining part of dimension is discarded. The actually correct implementation should look like following:

```js
canvas.width = imageDom.width;
canvas.height = imageDom.height;

const context = canvas.getContext("2d");
context.imageSmoothingQuality = "high";
context.imageSmoothingEnabled = true;

// the remaining part is not disordered
context.drawImage(imageDOM, 0, 0);
```

I wrote my own cli tool, got curious if there is already implementation, saw this, just giving a heads-up.

Here is an example json file.

[info.json.zip](https://github.com/shoushouminami/modelpress-downloader/files/6529617/info.json.zip)

**Comment by shoushouminami** [comment]
Hi @aihornmac , 

Thanks for pointing out! Didn't realize the remaining part is still valid data. Will fix in the next release. Good job on making a cli tool btw.

**Comment by shoushouminami** [comment]
closing as the fix has been released in v3.14
